app/src/main/java/ar/com/wolox/android/example/ui/customView/CustomToolbar.kt

```diff
@@ -0,0 +1,30 @@
+package ar.com.wolox.android.example.ui.customView
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import androidx.appcompat.widget.Toolbar
+import ar.com.wolox.android.R
+
+class CustomToolbar @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = R.attr.toolbarStyle
+) : Toolbar(context, attrs, defStyleAttr) {
+
+    private val woloxIcon = R.drawable.wolox_logo
+
+    init {
+        setBackgroundColor(Color.TRANSPARENT)
+        setLogo(woloxIcon)
+        setTitle(R.string.example_wolox)
+    }
+
+    fun setHomeTitle() {
```

Comment:
No, the `CustomView` should not be attached to all the views. If we have 100 views there will be 100 similar methods. It's not okey. 
The `CustomToolbar` will have always the title `Wolox` and then you add the actual page, like `Wolox - X` being X variable. So there should be a way to modify that X.